cypress/integration/dib-travel/sign-in/sign-in-agent.spec.ts

```diff
@@ -14,6 +14,15 @@ describe('Sign In (Agent)', () => {
     cy.get('new-login .auth-container-footer a[href="/login/agent"]').click();
   });
 
+  it('should check if a not logged agent can visit a page', () => {
+    cy.visit('https://develop--dib-travel.netlify.app/people-management/employees');
```

Comment:
I don't know why we need a complete path here, or we just can: `cy.visit('/people-management/employees');` ?